src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  width: 200px;
+
+  padding: 32px 16px 16px;
+
+  background-color: #{$color-white};
+  border: 1px solid #{$main-border-color};
+  border-radius: 5px;
+
+  &__image {
+    display: block;
+
+    width: 160px;
+    height: 134px;
+
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+
+    line-height: 18px;
```

Comment:
do line-height: 18px common for many classes

Reply:
I agree) But think it's better to make variable fo this)
Thanks for review)